lib/crit_tables/cold_critical_table.rb

```diff
@@ -341,7 +341,7 @@ module CritRanks
                                   :wound_rank      => 3,
                                   :secondary_wound => nil,
                                   :regex           =>
-                                                      /Icy blast to neck freezes .*?'s words in mid\-speech and leaves it speechless./ },
+                                                      /Icy blast to neck freezes .*? words in mid\-speech and leaves (?:her|him|it) speechless\.\.\.permanently./ },
```

Comment:
Why strike the `'s` from the regex?  It exists in 550 plus other locations, and works fine that way.  Seems to be needed here, too, to get just 'the triton warlock'.

Reply:
Icy blast to neck freezes <pushBold/>the <a exist="862580855" noun="warlock">triton warlock</a><popBold/>'s words in mid-speech and leaves <pushBold/><a exist="862580855" noun="warlock">her</a><popBold/> speechless...permanently!

mistake.